python/vTestInterface.py

```diff
@@ -560,7 +580,99 @@ def validateClicastCommand (command, mode):
             raise UsageError()
         
 
-def processCommand (mode, clicast, pathToUse, testString="") -> dict:
+def updatedEnvironment (enviroPath, jsonOptions):
+    """
+    pathToUse is the full path to the environment directory
+    jsonOptions has the new values of ENVIRO.* commands for the enviro script
+    e.g.  ENVIRO.COVERAGE_TYPE: Statement
+
```

Comment:
We are not editing the original `.env` at any point, it might make sense to point this out in the function description here